gzip_ecap_extended_compressible_types_v1_2.patch

```diff
@@ -0,0 +1,48 @@
+--- src/adapter_gzip.cc	Sun Jun 12 02:10:41 2016
++++ src/adapter_gzip.cc	Sun Jun 12 02:14:33 2016
+@@ -367,7 +367,8 @@
+ 
+ 	/**
+ 	 * Checks the Content-Type response header.
+-	 * At this time, only responses with "text/html" content-type are allowed to be compressed.
++	 * Added more real text types
++	 * Modified by Y.Voinov (c) 2016
+ 	 */
```

Comment:
You need to remove the lines, `commit --amend` and force push to your `ptach-1` branch.